js/extension/state/epics.js

```diff
@@ -1,37 +1,38 @@
-import { loadedSchemas /* ,loadError */ } from "./actions";
+import { loadedSchemas, loadError } from "./actions";
 import { layersSelector } from 'mapstore2/web/client/selectors/layers';
 import * as MapInfoUtils from 'mapstore2/web/client/utils/MapInfoUtils';
 import { ReportIdentifyViewer } from '../plugins/Extension.jsx';
 import { updateNode } from 'mapstore2/web/client/actions/layers';
 import { updateFeatureInfoClickPoint } from "@mapstore/actions/mapInfo";
 import Rx from "rxjs";
 
-import { mockSchemas } from "../plugins/mockSchemas";
+// import { mockSchemas } from "../plugins/mockSchemas";
 import '../assets/style.css';
-// import axios from '@mapstore/libs/ajax';
+import axios from '@mapstore/libs/ajax';
 
 
 export const fetchSchemasEpic = (action$) => action$.ofType('FETCH_SCHEMAS').switchMap(() => {
     // set custom viewer to inject extension component into Identify
     MapInfoUtils.setViewer('reportViewer', ReportIdentifyViewer);
 
     // TODO: remove mocks and use API response
-    return Rx.Observable.of(loadedSchemas(mockSchemas));
-
-    // // use axios in prod, fetch in dev
-    // // TODO: replace 'reports' API (used to test ping) by 'schemas' API once it exists
-    // const fetchAPI = process.env.NODE_ENV === 'production' ?
-    //     axios.get('../mapstore-reports/reports')
-    //         .then(response => response.data) :
-    //     fetch('https://georchestra.mydomain.org/mapstore-reports/reports')
-    //         .then(response => response.json());
-
-    // return Rx.Observable.fromPromise(fetchAPI)
-    //     .switchMap((response) => {
-    //         console.log('Fetched schemas: ', response)
-    //         return Rx.Observable.of(loadedSchemas(response))
-    //     })
-    //     .catch(e => Rx.Observable.of(loadError(e.message)));
+    // return Rx.Observable.of(loadedSchemas(mockSchemas));
```

Comment:
we may think to a way of being able to use mock/or local compo. But this it out of this PR scope.